broker-daemon/broker-rpc/broker-rpc-server.js

```diff
@@ -124,7 +124,7 @@ class BrokerRPCServer {
     this.adminService = new AdminService(this.protoPath, { logger, relayer, engines, orderbooks, store, auth: this.auth })
     this.server.addService(this.adminService.definition, this.adminService.implementation)
 
-    this.orderService = new OrderService(this.protoPath, { logger, blockOrderWorker, auth: this.auth })
+    this.orderService = new OrderService(this.protoPath, { logger, relayer, blockOrderWorker, auth: this.auth, store })
```

Comment:
I think `blockOrderWorker` is unnecessary

Reply:
This service also contains the block order worker endpoints so I think we need it to maintain the original functionality until we do the switch